src/pages/index.js

```diff
@@ -23,6 +24,7 @@ export default function Home({ boundary, blocks, ...props }) {
       <Hero {...heroProps} boundary={boundary} />
       <HowItWorks {...blocks?.howItWorks} />
       <DataVisuals {...blocks?.dataVisuals} />
+      <OurCourses {...blocks?.ourCourses} />
```

Comment:
Our Courses doesn't belong on the homepage.

Reply:
Fixed